### DISCUSSION TITLE
suggest adding verse range to BIBLE reference link

### DISCUSSION THREAD

**Discussion by seesmof** [tim-hub/obsidian-bible-reference#227] [Ideas]
GOD bless you guys, may JESUS CHRIST our HOLY LORD GOD MOST HIGH give you an amazingly blessed fruitful peaceful joyful day today and forevermore

JESUS showed me that bolls.life supports selecting verses, you just have to add them as another parameter after `/`, here's an example: for John 3:15-21 in BSB we have https://bolls.life/BSB/43/3/15-21/

it would be good if the link our plugin forms will add the verses range also. GOD bless

**Comment by tim-hub** [comment]
> GOD bless you guys, may JESUS CHRIST our HOLY LORD GOD MOST HIGH give you an amazingly blessed fruitful peaceful joyful day today and forevermore
> 
> JESUS showed me that bolls.life supports selecting verses, you just have to add them as another parameter after `/`, here's an example: for John 3:15-21 in BSB we have https://bolls.life/BSB/43/3/15-21/
> 
> it would be good if the link our plugin forms will add the verses range also. GOD bless



FYI, this feature has been supported for verse range.

For example,
++john1:1-10
